site/sfguides/src/ai_assistant_for_sales_calls/ai_assistant_for_sales_calls.md

```diff
@@ -0,0 +1,250 @@
+author: Sikha Das
+id: ai_assistant_for_sales_calls
+summary: This application provides a centralized, AI-driven approach to analyze sales calls to gain actionable insights, enhance decision-making, and drive better alignment between product and GTM strategies.
+categories: Getting-Started
+environments: web
+status: Published 
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues
+tags: Getting Started, Data Science, Data Engineering, Snowpark, Snowflake Cortex, Streamlit
+
+# AI Assistant for Sales Calls
```

Comment:
@sfc-gh-nijadhav Change the title to "Building AI Assistant using Snowflake Cortex in Snowflake Notebooks"